### PR TITLE
Updated About Us

### PR DESCRIPTION
**Pull request by Pragati-3003**
- Updated About us section

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -142,13 +149,21 @@ <h2 class="text-white">
...
-    </p>
+
+
+    <h4 class="text-white readm style="text-align: center;">
+      <strong>Also read our  <a href="./public/terms&conditions.html" style="color: lightsteelblue; text-decoration: underline;">Terms and Conditions</a></strong>
+    </h4>
+
+
+
+
```
Comment (rohanbabbar04) [inline about.html:163]: REMOVE THESE LINES 
MAKE THE CODE CLEAN

File: resource.css [hunk 2]
```diff
@@ -440,6 +463,11 @@ section h5 {
   margin-right: 4px;
   padding: 5px;
 }*/
+
+/* .imge{
+  margin: 5px;
+  box-shadow: 2px 2px 8px rgba(253, 208, 126, 0.342)
+} */
```
Comment (rohanbabbar04) [inline resource.css:470]: Remove these lines if not required

**Review by rohanbabbar04 — COMMENTED** [review]
Remove all the extra lines. 
Don't comment code lines, use comments only for describing

**Comment by Pragati-3003** [comment]
> Remove all the extra lines. Don't comment code lines, use comments only for describing

@rohanbabbar04  I have removed all the unnecessary comments and lines .Please check .

**Review by rohanbabbar04 — COMMENTED** [review]
- The background looks a little light
- Also increase the font size of text
- Also, reduce the in-out animation...

**Comment by Pragati-3003** [comment]
> * The background looks a little light
> * Also increase the font size of text
> * Also, reduce the in-out animation...

@rohanbabbar04 , I have modified  the page , please check.

**Comment by rohanbabbar04** [comment]
A plain brown backgound does'nt look appealing, i would say we should go for an image. And address other changes accordingly..

**Comment by Pragati-3003** [comment]
> A plain brown backgound does'nt look appealing, i would say we should go for an image. And address other changes accordingly..

okie.

**Comment by Pragati-3003** [comment]
> A plain brown backgound does'nt look appealing, i would say we should go for an image. And address other changes accordingly..

@rohanbabbar04 , modified, please check.

**Comment by rohanbabbar04** [comment]
Looks much better @Pragati-3003 
Thanks !

**Review by rohanbabbar04 — APPROVED** [review]

**Pull request merged**